server/service/todoService.js

```diff
@@ -0,0 +1,141 @@
+import TodoModel from "../model/todosModel.js";
+
+class TodoService {
+    async deleteTodo(req, res) {
```

Comment:
the request and response object shouldn't be here, only in the controllers...here you need only the todoId as parameter...apply this logic to each service function